doc/user/output.xml

```diff
@@ -135,8 +117,16 @@ Type: 'scons program' to build the production program,
 
     <para>
 
-    When used with &AddOption; Help("text", append=False) will clobber any help output associated with AddOption().
-    To preserve the help output from AddOption(), set append=True.
+    Calling <literal>Help("text")</literal> will clobber
```

Comment:
maybe `overwrite` instead of `clobber` ?

Reply:
I did push a rewrite, see if that's better.